### PR TITLE
updated silicon tracker geometry to LBL design with symmetric disks

### PR DESCRIPTION
**Pull request by ShujieL**
Vertex barrel: removed support cylinder outside the 2nd vertex layer, and connect vertex layers to cone with disks.
Silicon disks: adjusted z positions, and make the backward design symmetric to forward region. 5 disks on each side.

### CODE REVIEW

File: compact/tracker.xml [hunk 1]
```diff
@@ -32,8 +32,8 @@
 
     <comment> Service/Support setup </comment>
     <constant name="VertexSupportCyl1_rmin"          value="VertexBarrel_rmin + 2.1*cm"/>
-    <constant name="VertexSupportCyl2_rmin"          value="VertexBarrel_rmax"/>
-    <constant name="VertexSupportCylCF_thickness"    value="300.0*um" />
+    <constant name="VertexSupportCyl2_rmin"          value="VertexBarrel_rmax-0.3*cm"/>
```
Comment (wdconinc) [inline compact/tracker.xml:35]: 0.3 cm is magical constant. Needs naming?

Comment (ShujieL) [reply compact/tracker.xml]: done

**Review by wdconinc — APPROVED** [review]

File: compact/tracker/support_service_assembly.xml [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
-    <comment> Inner vertex Support </comment>
-    <constant name="VertexSupportCyl1_length"             value="2*VertexSupportCyl1_rmin/tan(TrackerPrimaryAngle)" />
+    <comment> Inner vertex Support</comment>
+    <comment> get rid of the inner vertex support barrel </comment>
+<!--     <constant name="VertexSupportCyl1_length"             value="2*VertexSupportCyl1_rmin/tan(TrackerPrimaryAngle)" />
     <constant name="VertexSupportCyl1_zmax"               value="VertexSupportCyl1_length/2." />
     <constant name="VertexSupportCyl1_thickness"          value="VertexSupportCylCF_thickness" />
-
-    <comment> Outer Vertex Support </comment>
+ -->
```
Comment (sly2j) [inline compact/tracker/support_service_assembly.xml:13]: You can just delete everything related to the inner cylinder if it's not needed

File: compact/tracker/support_service_assembly.xml [hunk 3]
```diff
@@ -91,14 +102,36 @@
...
-      <support type="Cylinder"
+<!--       <support type="Cylinder"
         name="VertexSupportCyl1"
         vis="TrackerSupportVis"
         rmin="VertexSupportCyl1_rmin"
         length="VertexSupportCyl1_length"
         thickness="VertexSupportCyl1_thickness">
           <position x="0*cm" y="0*cm" z="0*cm" />
           <component material="CarbonFiber" thickness="VertexSupportCylCF_thickness" name="Carbon Fiber Shell" vis="TrackerSupportVis"/>
+      </support> -->
```
Comment (sly2j) [inline compact/tracker/support_service_assembly.xml:113]: This too, you can just delete.

**Review by sly2j — CHANGES_REQUESTED** [review]
I'll approve and merge once you remove the dead code. I want to prevent having lots of commented out blocks, we can always add them back later if needed.

**Comment by wdconinc** [comment]
Yeah, I agree with @sly2j, we should just remove code instead of commenting it out. If it is commented out, it is not tested, not updated, and it might as well be not working or even wrong (and that's how it should be treated). So just remove the commented blocks entirely.

File: compact/tracker/support_service_assembly.xml (hunk 3 above)
Comment (ShujieL) [reply compact/tracker/support_service_assembly.xml]: done

**Review by wdconinc — APPROVED** [review]

**Pull request merged**